introduction.md

```diff
@@ -0,0 +1,6 @@
+Hello my name is Kok-Meng Lam.
```

Comment:
I've responded but did not add any path yet as I'm new to Code School and have yet to explore other features/apps.